README.md

```diff
@@ -162,14 +162,17 @@ in URL-encoded or base64 format, which can be indicated by prefixing the data as
 Allow use of the **@SQLCHECKBOX** action tag for SQL fields, which will replace the drop-down list
 with a checkbox for each option similar to a checkbox field.
 
-When writing an SQL query for use with the *@SQLCHECKBOX* action tag, ensure that the returned
+When writing an SQL query for use with the `@SQLCHECKBOX` action tag, ensure that the returned
 column names are `code` and `label`, otherwise the query will not function. Also note that the
-*@SQLCHECKBOX* action tag does not work with the *@IF* action tag.
+`@SQLCHECKBOX` action tag does not work with the `@IF` action tag.
 
 For best results, the SQL query should return all possible options outside of a record context<br>
 (where \[record-name\] = '') and only limit to a subset (if required) within a record context.
 
-*The previous implementation of @SQLCHECKBOX where the tag is applied to checkbox fields is now
+Note that for performance reasons, the number of options returned by an `@SQLCHECKBOX` field may be
```

Comment:
Perhaps some more explanation may be worthwhile here as to why there is a performance limit?  As it is dynamic, one can't necessarily know until 'run-time' how many rows it will return, but understanding the issue from README guidance will help developers.  How about...
Note the @SQLCHECKBOX tag will create a hidden drop list box on the instrument that contains every combination of the checkbox options.  This means when the number of options returned grows, the number of possible combinations created increases exponentially, leading to performance degradation.   The practical limit on this will vary according to system, but it is advised to avoid trying to dynamically create an @SQLCHECKBOX with anything more than 15 checkbox items.